observermedia/eight-day-week
# Issue status taxonomy labels added

### Description of the Change
Issue status taxonomy CRUD labels added.

- view_item
- add_or_remove_items
- name_field_description
- slug_field_description
- parent_field_description
- popular_items
- parent_item
- parent_item_colon
- no_terms
- back_to_items


Closes #85 

### How to test the Change

- Translate or edit these texts in code
- Check if the modified texts appear in Issue Status term edit screen.

### Changelog Entry

> Added - Issue Status taxonomy CRUD labels


### Credits

Props @jayedul 


### Checklist:
